backend.py

```diff
@@ -20,10 +20,25 @@ def __init__(self, direction, path, path_front, coordinates):
         self.lives = 3
         self.flags = 0
         self.damages = 4
-        self.inactive = False
+
+    @property
+    def inactive(self):
+        """
+        Create read-only attribute. Evaluate robot's coordinates and return
+        a boolean.
```

Comment:
Tady napiš přímo dokumentaci atributu, ne té funkce. Něco jako:

```
True if the robot is inactive (not on the game board).

All inactive robots are moved to coordinates -1, -1.
Return True if robot is on these coordinates.
```

To, že se atribut vytvoří, je zřejmé z dekorátoru.
To, jak to funguje vevnitř (tady konkrétně odkaz na dokumentaci `@property`) je dobré dát do normálního komentáře, ne do dokumentačního řetězce.